core/utils.go

```diff
@@ -23,3 +23,25 @@ func NextPowerOf2[T constraints.Integer](d T) T {
 	nextPower := math.Ceil(math.Log2(float64(d)))
 	return T(math.Pow(2.0, nextPower))
 }
+
+// PadToPowerOf2 pads a byte slice to the nearest power of 2 length by appending zeros
+func PadToPowerOf2(data []byte) []byte {
```

Comment:
@mooselumph can you confirm if this is the correct way to enforce the number of chunks to be power of 2?

Reply:
This is enforcing the number of bytes being a power of two rather than the number of chunks, which isn't what we would want.

I'm also not sure that such a function should be needed. We just need to make sure that the `Length` claimed by the client in the `BlobHeader` is appropriately rounded up to the next power of two.